app/gitcoinbot/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from gitcoinbot.models import GitcoinBotResponses
+
+
+class GitcoinBotResponsesAdmin(admin.ModelAdmin):
```

Comment:
Can you add a docstring for this module and class that adheres to PEP-257/PEP-258 here:

```python
# -*- coding: utf-8 -*-
"""Define Gitcoin Bot specific models."""
from django.contrib import admin

from gitcoinbot.models import GitcoinBotResponses


class GitcoinBotResponsesAdmin(admin.ModelAdmin):
    """Define the Gitcoin Bot response admin model for displaying bot request data."""

    fields = ('_request', 'response', )
    list_display =  ('_request', 'response', )


admin.site.register(GitcoinBotResponses, GitcoinBotResponsesAdmin)
```
Additionally, can you either remove the trailing commas from the variables below or add trailing whitespace between `,)` ?
`fields = ('_request', 'response', )` or `fields = ('_request', 'response')`